docs/modules/reference/pages/telemetryd/protocols/bmp/persisting-adapter.adoc

```diff
@@ -70,15 +74,16 @@ config:property-set rpkiUrl http://localhost:8080/api/export.json
 config:update
 ----
 
-== Run BMP Adapter on Sentinel
+[[run-bmp-sentinel]]
+== Run BMP adapter on Sentinel
 
-BMP Persisting Adapter can also run on Sentinel.
+The BMP persisting adapter can also run on Sentinel.
 Sentinel must start the appropriate BMP adapter.
 In Sentinel, adapters are configured either by placing a .cfg file in `$\{SENTINEL_HOME}/etc` or via a `config:edit` statement.
 
 The following example configures the consumption of `BMP` messages and saves the configuration in `$\{SENTINEL_HOME}/etc/org.opennms.features.telemetry.adapters-bmp.cfg`.
 
-First, login to the Karaf Shell.
+First, log in to the Karaf Shell.
 
 [source, console]
```

Comment:
@cgorantla Should we also add a note here about adding these features to featuresBoot.d on the Sentinel?

Reply:
@cgorantla what do you think about Mark's question?

Reply:
I think it makes sense to add that. I don't have right setup to validate this. 
So I would leave it to you guys to take care of it.

Reply:
We can handle that. Thanks for verifying.